src/NoteWorkspace.ts

```diff
@@ -74,7 +74,7 @@ export class NoteWorkspace {
   static _rxBeginTag = '(?<= |,|^)#'; // match # preceded by a space, comma, or newline, regardless of whether it is followed by a letter character
   static _rxWikiLink = '\\[\\[[^sep\\]]+(sep[^sep\\]]+)?\\]\\]'; // [[wiki-link-regex(|with potential pipe)?]] Note: "sep" will be replaced with pipedWikiLinksSeparator on compile
   static _rxTitle = '(?<=^( {0,3}#[^\\S\\r\\n]+)).+';
-  static _rxMarkdownWordPattern = '([_\\p{L}#\\.\\/\\\\]+)'; // had to add [".", "/", "\"] to get relative path completion working and ["#"] to get tag completion working
+  static _rxMarkdownWordPattern = '([_\\p{L}\\d#\\.\\/\\\\]+)'; // had to add [".", "/", "\"] to get relative path completion working and ["#"] to get tag completion working
```

Comment:
Last thing before merge. This looks like a mistake, maybe -- a random digit has been added to the markdown word pattern?

Reply:
@Anderas

Reply:
This was added on purpose: without it links and citations were highlighted (with cmd+hover) without the number part (eg only `author_title_` instead of `author_title_year`). And it seems that other links / references could also contain numbers to benefit from this.